Source/Our.Umbraco.Nexu.Core/Mapping/TypeConverters/RelationsToRelatedDocumentsConverter.cs

```diff
@@ -23,12 +25,18 @@ internal class RelationsToRelatedDocumentsConverter : ITypeConverter<IEnumerable
         /// </summary>
         private readonly IContentService contentService;
 
+        /// <summary>
+        /// The localization service.
+        /// </summary>
+        private readonly ILocalizationService localizationService;
```

Comment:
Is it not better to use UmbracoHelper to get dictionairy items because then the calls are cached ? Or does the ILocalizationService cache as well ?

Reply:
I don't know if the LocalizationService uses caching to be honest.
But I guess it's not that big of a deal, because it's a back-office request instead of front-end (low traffic).
You're welcome to switch to an UmbracoHelper instance though :-)